src/components/transaction/GetAllTransactions.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import getRequestStatus from 'utils/getRequestStatus'
+
+const GetAllTransactions = () => {
+  const [transactions, setTransactions] = useState<Array<TransactionResponse>>()
+
+  const [requestStatus, setRequestStatus] = useState<
+    'fetching' | 'error' | 'idle' | 'success'
+  >('idle')
+
+  const [errorMessage, setErrorMessage] = useState(null)
+
+  return (
+    <div className="sectionContainer">
+      <h2>View all transactions</h2>
+      <button
+        className="getTransactionsButton"
+        onClick={() => {
+          setRequestStatus('fetching')
+          fetch('https://nestjs-bank-app.herokuapp.com/transactions')
+            .then((res) => {
+              if (!res.ok) {
+                setRequestStatus('error')
+                throw Error(res.statusText)
+              }
+              setRequestStatus('success')
+              return res.json()
+            })
+            .then((data) => setTransactions(data))
+            .catch((e) => setErrorMessage(e.message))
+        }}
+      >
+        Get transactions
+      </button>
+      <div className="requestStatus">
+        {getRequestStatus(requestStatus, errorMessage)}
+      </div>
+      <div>
+        {transactions && requestStatus === 'success' ? (
+          <>
+            {transactions?.map((transaction, i) => (
```

Comment:
You can simplify this ever so slightly. There is no need to wrap the map in a fragment here
```suggestion
          transactions?.map((transaction, i) => (
```